wordpress-mobile/WordPress-iOS
# Add email followers filter to people page

See #6969

Adds an email filter to the people page.

**Related PRs:** 

- wordpress-mobile/WordPressKit-iOS#477
- wordpress-mobile/WordPressKit-iOS#480

### To test:

- Login to an account with email followers (or invite one)
- On the site tab, scroll down to Configure and tap on People
- Tap on "Email Followers" to see your site's email followers
- Tap on a email follower and view the email follower's page
- Remove the follower if you would like to test the removal

## Screenshots
| | |
|-|-|
| <img width="490" alt="Screen Shot 2022-01-26 at 8 44 46 PM" src="https://user-images.githubusercontent.com/2454408/151284932-e9f04b92-f185-428c-9289-e7b4a52f24be.png"> | <img width="490" alt="Screen Shot 2022-01-26 at 8 54 52 PM" src="https://user-images.githubusercontent.com/2454408/151284936-63756514-49f0-45eb-a988-7d25a2600e37.png"> |
| <img width="490" alt="Screen Shot 2022-01-26 at 8 54 57 PM" src="https://user-images.githubusercontent.com/2454408/151284939-8f4e57c2-b9fb-41a3-84b4-4705dff054c8.png"> | <img width="490" alt="Screen Shot 2022-01-26 at 8 55 04 PM" src="https://user-images.githubusercontent.com/2454408/151284940-393effe5-d301-42e1-ac3e-1017501dc176.png"> |

## Regression Notes
1. Potential unintended areas of impact
Potentially the other filters

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Verified those filters were still working correctly after adding the email filter.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
